.github/workflows/deploy_main.yml

```diff
@@ -79,11 +83,13 @@ jobs:
       - name: Invalidate cache on AWS CDNs
         uses: chetan/invalidate-cloudfront-action@master
         env:
-          DISTRIBUTION: "ED03LPVC4OXSW" # TODO: update to new CDN after access is provided
+          DISTRIBUTION: ${{env.WEB_CDN_ID}}
           PATHS: "/*"
           AWS_REGION: "us-east-1"
           AWS_ACCESS_KEY_ID: ${{ secrets.CLIENT_DEV_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.CLIENT_DEV_AWS_SECRET_ACCESS_KEY }}
+      - name: Remove staging artifacts
+        run: aws s3 rm s3://usds-geoplatform-justice40-website/justice40-tool/ --exclude "*" --include "$(echo $PR_NUMBER)-*" --recursive
```

Comment:
So `--include "$(echo $PR_NUMBER)-*" --recursive` means that the only objects that will be removed are only any files and directories that have the `$PR_NUMBER` in the path name? What exactly does `--exclude "*"` do?

Reply:
I'm placing the exclude first meaning, first exclude everything. Then include only the artifacts with the PR prefix. It's an added safety check to ensure that only the folder with the PR prefix get's removed. A --dryrun flag was ran and it seemed to work as expected.

Reply:
It looks like maybe this would delete folders for pull requests that end with a given number, not match a given number. For example if a pull request number is 122, this code would delete a folder with 122- AND 1122- in the name, right? Which I don't think we'd want, even though it seems like an unlikely edge case.

Reply:
@switzersc-usds : I don't think it would delete the folder with 1122, to be sure, ran a quick test:

Created both folder prefixes, 3 folders (1 sub-folder) with a 111 prefix and one with 2111 prefix:
![Screen Shot 2021-10-05 at 12 23 02 PM](https://user-images.githubusercontent.com/86254807/136089581-c9dec6c0-93ee-4814-9d22-10a71de96e2f.png)

Then ran the delete command twice:

[Run 1 ](https://github.com/usds/justice40-tool/runs/3806950036?check_suite_focus=true): PR set to 2111 - Expecting only 1 folder to be removed:

![Screen Shot 2021-10-05 at 12 27 09 PM](https://user-images.githubusercontent.com/86254807/136089967-e7f46ce0-491c-4e1e-b1d2-c0ba14aff22f.png)

Meets expectation to delete one and only 1 folder

[Run 2](https://github.com/usds/justice40-tool/runs/3807086047?check_suite_focus=true): PR set to 111 - Expecting 3 folders (and 1 sub-folder) to be removed:
![Screen Shot 2021-10-05 at 12 29 56 PM](https://user-images.githubusercontent.com/86254807/136091148-a9f4ff7f-42e4-4bce-a3a0-88e8e50e91b2.png)

Meets expectation to delete 3 and only 3 folders (and 1 sub-folder).

Reply:
Oh great, thanks!